From 60c9a415a60efd7dea68b73a91fa1688256142dc Mon Sep 17 00:00:00 2001
From: Kilyoung Shin <gilbert.shin@samsung.com>
Date: Thu, 28 Oct 2021 10:21:46 -0700
Subject: [PATCH 1/2] Grant permission for mediatranscoding hal_allocator for
 GSI image

resolved compilation issue addin same in
prebuilts/api/31.0/private/mediatranscoding.te

Bug: 203839961
Test: run cts -m CtsMediaTranscodingTestCases -t
android.media.mediatranscoding.cts.MediaTranscodingManagerTest#testAddingClientUids

Signed-off-by: Kilyoung Shin <gilbert.shin@samsung.com>
Change-Id: If44653f436d4e5dcbd040af24f03b09ae8e7ac05
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>

diff --git a/prebuilts/api/31.0/private/mediatranscoding.te b/prebuilts/api/31.0/private/mediatranscoding.te
index 2a43cf9b5..073e81d78 100644
--- a/prebuilts/api/31.0/private/mediatranscoding.te
+++ b/prebuilts/api/31.0/private/mediatranscoding.te
@@ -19,6 +19,7 @@ hal_client_domain(mediatranscoding, hal_graphics_allocator)
 hal_client_domain(mediatranscoding, hal_configstore)
 hal_client_domain(mediatranscoding, hal_omx)
 hal_client_domain(mediatranscoding, hal_codec2)
+hal_client_domain(mediatranscoding, hal_allocator)
 
 allow mediatranscoding mediaserver_service:service_manager find;
 allow mediatranscoding mediametrics_service:service_manager find;
diff --git a/private/mediatranscoding.te b/private/mediatranscoding.te
index 2a43cf9b5..073e81d78 100644
--- a/private/mediatranscoding.te
+++ b/private/mediatranscoding.te
@@ -19,6 +19,7 @@ hal_client_domain(mediatranscoding, hal_graphics_allocator)
 hal_client_domain(mediatranscoding, hal_configstore)
 hal_client_domain(mediatranscoding, hal_omx)
 hal_client_domain(mediatranscoding, hal_codec2)
+hal_client_domain(mediatranscoding, hal_allocator)
 
 allow mediatranscoding mediaserver_service:service_manager find;
 allow mediatranscoding mediametrics_service:service_manager find;
-- 
2.38.1

